azure-pipelines.yml

```diff
@@ -41,7 +41,7 @@ stages:
 
     - script: |
         echo 'Downloading AppInsights Agent'
-        wget https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.14/applicationinsights-agent-3.4.14.jar \
+        wget https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.16/applicationinsights-agent-3.4.16.jar \
```

Comment:
In the near future if we have cycles, we should look into defining variables for values that require changes/updates